Algorithm-Caesars-Cipher.md

```diff
@@ -72,21 +129,16 @@ function rot13(str) {
 }
 ```
 
-# Code Explanation:
-#### First solution
-- A string variable `nstr` is declared and initialized to store the
-decoded string.
-- The for loop is used to loop through each character of the input string.
-- If the character is not uppercase English alphabets(i.e. its ascii doesn't lie between 65 and 91 ), we'll leave it
-as it is and [continue](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue) with next iteration.
-- If it's the uppercase English alphabet, we'll subtract 13 from it's
-ascii code.
-- If the ascii code is less than 78, it'll get out of
-range when subtracted by 13 so we'll add 26(number of letters in
-        English alphabets) to it so that after A it'll go back to Z.
-e.g. M(77) ↔ 77-13 = 64(Not an English alphabet) +26 = 90 ↔ Z(90)
 
-# Credits:
-If you found this page useful, you can give thanks by copying and pasting this on the main chat:  **`thanks @anuragaryan and @SaintPeter for your help with Algorithm: Caesar's Cipher`**
+### :trophy: Credits:
+If you found this page useful, you may say thanks to the contributors by copying and pasting the following line in the main chat:
+
+**`thanks @anuragaryan @SaintPeter @vaskezu for your help with Algorithm: Caesar's Cipher`**
+
+## :clipboard: NOTE TO CONTRIBUTORS:
+- :warning: **DO NOT** add solutions that are similar to any existing solutions. If you think it is ***similar but better***, then try to merge (or replace) the existing similar solution.
+- Add an explanation of your solution.
+- Categorize the solution in one of the following categories &mdash; **Basic**, **Intermediate** and **Advanced**. :traffic_light:
+- Please add your username only if you have added any **relevant main contents**. (:warning: ***DO NOT*** *remove any existing usernames*)
 
-> **NOTE:** Please add your username only if you have added any **relevant main contents** to the wiki page. (Please don't remove any existing usernames.)
+> See :point_right: [**`Challenge Solution Template`**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Challenge-Solution-Template) for reference.
```

Comment:
See :point_right: [**`Challenge Solution Template`**](Challenge-Solution-Template) for reference.